hosted_control_planes/index.adoc

```diff
@@ -9,6 +9,12 @@ You can deploy {product-title} clusters by using two different control plane con
 toc::[]
 
 include::modules/hosted-control-planes-overview.adoc[leveloffset=+1]
+include::modules/hcp-ocp-differences.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../security/encrypting-etcd.adoc#encrypting-etcd[enable etcd encryption]
```

Comment:
Should this link be a bit more embedded and link to Enabling etcd encryption section? The link metadata should then state **Enabling etcd encryption**.